### PR TITLE
Dop/update sample issues

### PR DESCRIPTION
**Pull request by doneill**

### CODE REVIEW

**Comment by doneill** [comment]
@shellygill Please review for beta-dev branch inclusion.

File: identify-graphics-hittest/src/main/java/com/esri/arcgisruntime/sample/identifygraphicoverlay/MainActivity.java [hunk 1]
```diff
@@ -70,45 +63,22 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        polygonGeometry.addPoint(20e5, 20.e5);
+        polygonGeometry.addPoint(20e5, -20e5);
+        polygonGeometry.addPoint(-20e5, -20e5);
 
         // create solid line symbol
-        int blue = Color.rgb(0, 0, 230);
-        SimpleLineSymbol solidLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, blue, 4);
-        // create graphic from polygon and symbol
-        Graphic graphic = new Graphic(polygon, solidLine);
+        int yellow = Color.argb(4, 255, 255, 0);
```
Comment (shellygill) [inline identify-graphics-hittest/src/main/java/com/esri/arcgisruntime/sample/identifygraphicoverlay/MainActivity.java]: This is specifying an almost-completely-transparent fill symbol. Suggest Color.argb(255, 255, 255, 0) instead (or at least 128 for the alpha component).

File: identify-graphics-hittest/src/main/java/com/esri/arcgisruntime/sample/identifygraphicoverlay/MainActivity.java [hunk 2]
```diff
@@ -118,28 +88,7 @@ private void addGraphicsOverlay(Point center) {
...
-        int id = item.getItemId();
-
-        //noinspection SimplifiableIfStatement
-        if (id == R.id.action_settings) {
-            return true;
-        }
-
-        return super.onOptionsItemSelected(item);
+        Log.i(TAG, "Graphic Overlay added to MapView");
     }
```
Comment (shellygill) [inline identify-graphics-hittest/src/main/java/com/esri/arcgisruntime/sample/identifygraphicoverlay/MainActivity.java]: Remove logging (presume this was a temporary addition)?

**Comment by shellygill** [comment]
@doneill Couple of minor comments, back to you.

**Comment by doneill** [comment]
Thanks @shellygill updated with your comments and am merging.

**Pull request merged**